recipes/pysyntect/build.sh

```diff
@@ -0,0 +1,24 @@
+set -ex
+
+# Set conda CC as custom CC in Rust
+export CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER=$CC
+
+# Print Rust version
+rustc --version
+
+# Install cargo-license
+cargo install cargo-license
```

Comment:
Can you install this into `$BUILD_PREFIX` instead of `$HOME/.cargo/bin`?

Reply:
Let me check